ogr/ogrutils.cpp

```diff
@@ -103,37 +102,50 @@ std::string roundup(std::string s)
     }
     if (negative)
         s = '-' + s;
-    return s;
 }
 
 // This attempts to eliminate what is likely binary -> decimal representation
 // error or the result of low-order rounding with calculations.  The result
 // may be more visually pleasing and takes up fewer places.
-std::string intelliround(std::string &s)
+void intelliround(std::string &s)
 {
+    const size_t len = s.size();
+
+    // If we don't have ten characters, don't do anything.
+    if (len <= 10)
```

Comment:
If there is any chance we might revisit this literal in the future I'd really like it to be a named constant.

Reply:
done